lib/capify-ec2.rb

```diff
@@ -347,16 +347,47 @@ def deregister_instance_from_target_groups_by_dns(server_dns, target_group_names
       })['target_groups'][0]['target_group_arn']
 
       # Deregister the instance with the target group ARN
-      result = @alb_client.deregister_targets({
+      @alb_client.deregister_targets({
         target_group_arn: target_group_arn,
         targets: [ { id: instance.id } ]
       })
 
-      raise "Unable to remove instance from ELB '#{load_balancer.id}'..." unless result#successful
+      # Loop until instance is deregistered or timeout is reached
+      begin
+        Timeout::timeout(options[:timeout]) do
```

Comment:
I don't think options exists here does it? I think this was one of the problems I had last night.

Reply:
I think we should add `:timeout` to the ruby script rather than hardcode here. It's difficult to determine what an appropriate value would be here.

Reply:
Agreed. I'll temporarily set it at 5 - can you figure out what the best way to do what you suggested is? I'm not sure what the best approach is?